### PR TITLE
1687, 1688 tweak multiple disease type/primary site 

### PR DESCRIPTION
**Pull request by cy**
screen shot 2017-09-28 at 12 14 29 pm

### CODE REVIEW

File: src/packages/@ncigdc/tableModels/projectsTableModel.js [hunk 1]
```diff
@@ -79,21 +79,23 @@ const projectsTableModel = [
...
-                  expanded
-                    ? <span>{`\u25B4 less`}</span>
-                    : <span>{`\u25BE Show ${node.disease_type
-                        .length} items`}</span>,
-              })}
-        />
+      <Td
+        key={node.disease_type}
+        style={{
+          width: '200px',
```
Comment (Jephuff) [inline src/packages/@ncigdc/tableModels/projectsTableModel.js]: Maybe max width instead?

File: src/packages/@ncigdc/tableModels/projectsTableModel.js [hunk 2]
```diff
@@ -103,21 +105,20 @@ const projectsTableModel = [
...
-                ToggleText: ({ expanded }) =>
-                  expanded
-                    ? <span>{`\u25B4 less`}</span>
-                    : <span>{`\u25BE Show ${node.primary_site
-                        .length} items`}</span>,
-              })}
-        />
+      <Td
+        key="primary_site"
+        style={{ maxWidth: '200px', padding: '3px 15px 3px 3px' }}
```
Comment (Jephuff) [inline src/packages/@ncigdc/tableModels/projectsTableModel.js]: hmm, the text isn't wrapping, do you need whitespace normal here?

Comment (cy) [reply src/packages/@ncigdc/tableModels/projectsTableModel.js]: screen shot 2017-09-28 at 2 28 17 pm
isn't it? I'm looking at the wrong spot?

Comment (Jephuff) [reply src/packages/@ncigdc/tableModels/projectsTableModel.js]: oh hmmm, maybe I didn't have the up to date code, I will test again!

Comment (Jephuff) [reply src/packages/@ncigdc/tableModels/projectsTableModel.js]: Ahh I see, I am looking at a single value, so it's not in the collapsed list. so the single value needs it I think?
image

Comment (cy) [reply src/packages/@ncigdc/tableModels/projectsTableModel.js]: ohhhh

Comment (cy) [reply src/packages/@ncigdc/tableModels/projectsTableModel.js]: fixed, thank you 😊

**Pull request merged**